File: demo-vk-rest-scraping.java/src/main/java/demo/model/ItemsList.java
package demo.model;

import com.google.api.client.util.Key;
import demo.reader.RepeatableReader;

import java.util.List;

public class ItemsList implements RepeatableReader.Responsive<List<Item>> {

    @Key("response")
    private List<Item> response;

    @Override
    public List<Item> response() {
        return response;
    }

    public void setResponse(List<Item> response) {
        this.response = response;
    }

    @Override
    public String toString() {
        return "ItemsList{" + "response.size=" + (response != null ? response.size() : "?") + '}';
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/model/ItemsList.kt
package demo.model

import com.google.api.client.util.Key
import demo.reader.RepeatableReader

class ItemsList : RepeatableReader.Responsive<List<Item>> {

    @Key("response")
    private var response: List<Item>? = null

    override fun response(): List<Item>? {
        return response
    }

    override fun toString(): String {
        return "ItemsList{" + "response.size=" + (if (response != null) response!!.size else "?") + '}'
    }
}
